src/main/java/com/sarapis/orservice/controller/ServiceAreaController.java

```diff
@@ -0,0 +1,56 @@
+package com.sarapis.orservice.controller;
```

Comment:
Can you remove the `.DS_Store`. These are binary files autogenerated by your OS.